imports/ts-node.js

```diff
@@ -0,0 +1,3 @@
+import { register } from 'node:module'
+import { pathToFileURL } from 'node:url'
+register('ts-node/esm', pathToFileURL('./'))
```

Comment:
@mcollina 

I have a warning from node when I try to use experimental loader.
The warning message advise to use this code instead, how would you use ts-node as part of a node command?
Example:
```
node --import ./imports/ts-node.js --env-file=.env ./scripts/seed-database.ts
```

Reply:
Yea it used to be a big issue actually.. I needed to apply various workarounds in order to get loader working in Fastify:

```sh
FASTIFY_AUTOLOAD_TYPESCRIPT=1 VITEST=true node --no-warnings=ExperimentalWarning --loader ts-node/esm src/index.ts
```

I would like to see official support for `ts-node/esm`.

Reply:
Update: Ow it seems that `FASTIFY_AUTOLOAD_TYPESCRIPT=1 VITEST=true ` is no longer required anymore to get it working!? That is cool!

Reply:
>  I'm now sponsoring you via GitHub!

I don't even notice it is available now. Applied long time ago and forgotten about it.
Thanks for your sponsor.

> Why wouldn't you recommend ts-node/esm?

I am also using `ts-node/esm` in my environment, but it seems like the loader do not get much attention from the `TypeScript` team.